### ISSUE TITLE
Shortcut for autotyping only the password

### ISSUE THREAD

**Issue by jakabk** [keeweb/keeweb#1890]
**Is your feature request related to a problem? Please describe.**
I cannot use keeweb without a mouse because I cannot set a shortcut for auto-typing only the password (username handled other way)

**Describe the solution you'd like**
I would like to be able to set a different keystroke for auto-typing the password of the selected item.

**Comment by bdeeming** [comment]
Indeed, I've hit this on occasion also (not for a lack of a mouse, but just a desire to use a keyboard shortcut). What I find myself subconsciously doing (after searching for an entry) is trying to use the same shortcuts as in the auto-type dialog view (ie SHIFT+ENTER, CTRL+ENTER, etc).
To provide a little extra context: If I don't already have KeeWeb open then naturally the global auto-type shortcut doesn't work, so I also have a separate OS keyboard shortcut setup to open KeeWeb normally, then -> unlock DB -> type entry name -> SHIFT+ENTER... oh wait -> mouse -> double click.
Big thanks for the great app, and your amazing responsiveness to issue reports - I have followed the KeeWeb project issue tracker for some years now, so have observed a lot of dedication!
